test/common/config/subscription_test_harness.h

```diff
@@ -36,9 +36,10 @@ class SubscriptionTestHarness {
    * Expect that an update request is sent by the Subscription implementation.
    * @param cluster_names cluster names to expect in the request.
    * @param version version_info to expect in the request.
+   * @param first_on_stream whether the node information should be expected
```

Comment:
I think it would be clearer to rename this to `expect_node_message` to reflect its actual function (rather than intended use).

Reply:
Done